gltf/gltfpack.cpp

```diff
@@ -661,10 +678,39 @@ static void process(cgltf_data* data, const char* input_path, const char* output
 	    {"KHR_lights_punctual", data->lights_count > 0, false},
 	    {"KHR_texture_basisu", !json_textures.empty() && settings.texture_ktx2, true},
 	    {"EXT_mesh_gpu_instancing", ext_instancing, true},
+	    {"VRM", settings.vrm, false},
 	};
 
 	writeExtensions(json, extensions, sizeof(extensions) / sizeof(extensions[0]));
 
+	comma(json_extensions);
+	append(json_extensions, "\"extensions\":{");
+	if (settings.vrm)
+	{
+
+		std::string extension_vrm;
+		extension_vrm.assign("VRM");
+		for (size_t i = 0; i < data->data_extensions_count; ++i)
+		{
+			cgltf_extension extension = data->data_extensions[i];
+			if (!extension.name)
```

Comment:
This & above & below can be replaced with `if (strcmp(extension.name, "VRM") == 0)` (extension.name is guaranteed to be non-null)